### PR TITLE
Setup Android in Playground

### LINKED ISSUE #67: Create playground modules
IMHO we should have at least 1 playground module for each "module type" we plan to have in production.
We can start with the followings:

- `:playground:multiplatform`
- `:playground:androidapp`

### PR DESCRIPTION
**Pull request by 4face-studi0**
This PR creates 2 sub-modules in `playground`:
* Android
* Multiplatform

Android module contains a simple Application and Activity (Compose)

Also, a small bug has been fixed:
Inside the `plugin` block of Android plugins' `build.gradle` file we used `compose`, but it seems like `compose` is a existent function in Gradle, so it has been replaced by `composePlugin`

Closes #67

### CODE REVIEW

File: build-tools/android/build.gradle [hunk 1]
```diff
@@ -10,7 +10,7 @@ gradlePlugin {
             id = 'aditogether.android.app'
             implementationClass = 'aditogether.buildtools.android.AndroidAppPlugin'
         }
-        compose {
+        composePlugin {
```
Comment (fondesa) [inline build-tools/android/build.gradle:13]: All the `NamedDomainObjectContainer`, like this one, are translating the DSL key (e.g. `composePlugin`) to `register("composePlugin")`.
You don't need to change the name to avoid clashes, you can use:
```
register("compose") {
  ...
}
```
Instead of:
```
compose {
  ...
}
```

File: playground/android/src/main/kotlin/aditogether/playground/android/PlaygroundActivity.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.material3.Text
+
+class PlaygroundActivity : ComponentActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent { Text(text = "Hello ADI") }
```
Comment (fondesa) [inline playground/android/src/main/kotlin/aditogether/playground/android/PlaygroundActivity.kt:12]: The best UI

File: build-tools/android/build.gradle (hunk 1 above)
Comment (4face-studi0) [reply build-tools/android/build.gradle:13]: I wanted to avoid a different pattern. I was actually thinking about renaming them all as `xPlugin`. What do you think?

Comment (fondesa) [reply build-tools/android/build.gradle:13]: Fine for me! Also, if you want to align them, can you also remove `displayName` and `description` from some of the declarations since they are not useful unless we publish them on Maven?

Comment (4face-studi0) [reply build-tools/android/build.gradle:13]: I pushed the change.

I also did another change: the IDE shows a warning for some dependency notations from Version Catalog: e.g. if we have `libs.something` and `libs.something.sub`, there is a warning when using `libs.something`. Not sure if that's expected or a bug, but it seems a legit scenario to me. Have you got more info about it?

Comment (fondesa) [reply build-tools/android/build.gradle:13]: I've not any additional insights on IDE warnings, but in general, we should avoid having `libs.something.sub` and `libs.something` IMHO because it's very confusing.

Usually, in Gradle catalogs, every part between dots is considered a group.
Following this idea, if `libs.something` and `libs.something.sub` exist, I wouldn't understand what's the content of `libs.something`, should it contain `libs.something.sub` or not?

TL;DR don't know what this IDE warning is, but IMHO we shouldn't have `libs.something` and `libs.something.sub` anyway

Comment (4face-studi0) [reply build-tools/android/build.gradle:13]: The IDE it's warning about something like the usage of a group, indeed. In this case `libs.gradle.kotlin`, because we also have `libs.gradle.kotlin.serialization`

Comment (fondesa) [reply build-tools/android/build.gradle:13]: (anyway, I think we can also fix/change this in another PR in case)

Comment (fondesa) [reply build-tools/android/build.gradle:13]: IMHO for the moment we can just remove the serialization plugin since we are not using it. In future we can think about this problem if we have other warnings. What do you think?

**Review by fondesa — APPROVED** [review]

File: build-tools/android/build.gradle (hunk 1 above)
Comment (4face-studi0) [reply build-tools/android/build.gradle:13]: Fair enough, done

**Review by fondesa — APPROVED** [review]
